.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
> That seems too easy to get wrong by accident.

That would definitely belong in a git alias so you wouldn't actually type the whole thing :).